src/parse/imports.js

```diff
@@ -42,9 +42,21 @@ function imports(archive, result) {
     if (result.flow.find(MimeBuilderFlowItem)) {
       result.imports.MimeBuilder = true
     }
+
+    if (webSocketItem(entries)) {
+      result.imports.websocket = true
+    }
   }
 }
 
+function webSocketItem(entries) {
+  // This function checkes to see if the request is a ws or wss
+  //  request to make sure imports has websocket true
+  const check = ({ request }) =>
+    ['ws:', 'wss'].includes(request.url.slice(0, 3))
```

Comment:
nit: Since we do this twice, would it make sense to make an `isWebsocket` function on `request`?

Reply:
Ok